exporter/cassandraexporter/metadata.yaml

```diff
@@ -8,3 +8,6 @@ status:
   codeowners:
     active: [atoulme, emreyalvac]
 
+tests:
+  config:
+  skip_lifecycle: true
```

Comment:
Should they still be skipped or does this PR make it so this can be enabled?

Reply:
this PR doesn't allow them to be enabled yet.